Update sha256 checksum for 0.19.4 release

rickypai opened the pull request:
README has sha256 from 0.19.3 release. sha256 gotten from https://github.com/bazelbuild/rules_go/releases/tag/0.19.4

Fixes #2213

ianthehat commented:
Sorry about that.
There are two other occurrences, could you fix those as well please?
Also mention in the change description that it Fixes #2213

rickypai commented:
Updated PR to fix more occurrences: https://github.com/bazelbuild/rules_go/pull/2214/commits/3680cf18bfdffd75380e47bf001bfe9d0915719a

and added description
